### ISSUE TITLE
Consider including some form of cross-validation with subsampling routines (such as alpha estimating)

### ISSUE THREAD

**Issue by ssadjina** [ssadjina/FatTailedTools#4]
As a start, how would one go about doing this with `fit_alpha_and_scale_linear_subsampling`? Here's how it could work:
- Save the data that is not used during a "bootstrapped estimation" as a test set.
- Estimate parameters as usual with the bootstrapped sample.
- Estimate again with the test set.

**Comment by ssadjina** [comment]
Two questions:
1. Should one repeat the entire estimation with the test set _exactly_ as with the "train" set? Or should one, for example, estimate using the best threshold value? Or one could also calculate the MSE for how well the best estimated fit (threshold and alpha) works on the train set.
2. Can we even get help in estimating the correct parameters using a dedicated "validation" set, exactly as with ML-style cross-validation (use a train-validation split to establish the best hyperparameters and then test on test set)?
